src/main/java/com/fundation/webservice/controller/Controller.java

```diff
@@ -516,8 +517,8 @@ public AudioResponse convertAudio(@RequestParam("asset") MultipartFile asset, @R
             String outputChecksumString = "";
             try {
                 outputChecksumString = checksum.getChecksum(inputJson.getString("destPath") +
```

Comment:
this line must be complete 120 characters

Reply:
That was resolved.